### DISCUSSION TITLE
Comment in globals.css break the following `@import`

### DISCUSSION THREAD

**Discussion by ngotrongphuc** [tailwindlabs/tailwindcss#15446] [Help]
**What version of Tailwind CSS are you using?**

v3.4.9

**What build tool (or framework if it abstracts the build tool) are you using?**

postcss 8.4.47, Next.js 14.2.11

**What version of Node.js are you using?**

v20.17.0

**What browser are you using?**

Chrome

**What operating system are you using?**

macOS

**Reproduction URL**

**Describe your issue**

In `globals.css` if I put a comment before any import like, for example:
```
/* blabla */
@import url('https://fonts.googleapis.com/css2?family=Doto:wght@100..900&display=swap');
```
so the import line won't work. If I remove the comment line or move it after the import line, everything works fine.

**Comment by philipp-spiess** [comment]
Hey! In version 3, Tailwind CSS does not handle CSS imports specially in any way, you might need to add [`postcss-import`](https://www.npmjs.com/package/postcss-import) if you're seeing issues there!
